test/test_metrics.py

```diff
@@ -185,6 +185,16 @@ def test_execute_time_metric(self):
     # of `ExecuteComputation`, but the actual async time.
     self.assertGreater(execute_time_ns, .5 * wall_time_ns)
 
+  def test_transfer_to_server_time(self):
```

Comment:
Does this test case work on GPU/TPU? I'm asking because we added a skip for the execute time metric test when using CPU.

Reply:
@JackCaoG Confirmed that it works across GPU&TPU